ic/agent.py

```diff
@@ -113,6 +99,11 @@ def read_state_raw(self, canister_id, paths):
         }
         _, data = sign_request(req, self.identity)
         ret = self.read_state_endpoint(canister_id, data)
+        if ret == b'Invalid path requested.':
+            raise ValueError('Invalid path requested!')
+        elif ret == b'Could not parse body as read request: invalid type: byte array, expected a sequence':
+            raise ValueError('Could not parse body as read request: invalid type: byte array, expected a sequence')
+        print(ret)

```

Comment:
maybe remove this debug print

Reply:
if user input wrong paths, those error info could prompt them

Reply:
I mean the `print` in line 106

Reply:
yeah, I delete it now.